test/e2e/BrsComponents.test.js

```diff
@@ -719,4 +722,22 @@ describe("end to end brightscript functions", () => {
             "",
         ]);
     });
+
+    test("components/roAppInfo.brs", async () => {
+        manifest = preprocessor.getManifestSync(
+            process.cwd() + "/test/e2e/resources/conditional-compilation"
+        );
+        RoAppInfo.manifest = manifest;
```

Comment:
Watch out!  Since this value is never reset, any test that requires an accurate `RoAppInfo.manifest` after this will fail in a very unexpected way.  There's likely a way to solve this problem without explicitly reading an arbitrary manifest as part of this test (more details in a separate comment).

Reply:
Removed